services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -226,6 +228,7 @@ private void doDiscoverPathCommand(CommandData data) {
     private void doProcessIngressFlow(final CommandMessage message, String replyToTopic, Destination replyDestination)
             throws FlowCommandException {
         InstallIngressFlow command = (InstallIngressFlow) message.getData();
+        logger.info("Installing ingress flow {} on switch {}", command.getId(), command.getSwitchId());
```

Comment:
Ingress and one switch flow use one message format, but egress and transit - another.